README.md

```diff
@@ -298,8 +298,7 @@
        {day, ""} <- Integer.parse(day) do
     new(year, month, day)
   else
-    _ ->
-      {:error, :invalid_format}
+    _ -> {:error, :invalid_format}
```

Comment:
My bad. I realized there's no rule for the `->` syntax. I thought the extra line break after `->` looks odd since I was referring to the "convention" used for the case statement [here](https://github.com/lexmag/elixir-style-guide#multi-line-expr-assignment). Anyway, closing this PR. It seems my corrections are redundant. 😅